### PR TITLE
feat: add groups column to members page in organizations

### LINKED ISSUE #14315: Organization members page should have a "Groups" column that quickly shows their groups
The `/users` page has this today, we need to add this feature to the `/organization/{org-name}/members` page. The groups shown will be the groups in the selected organization.

Screenshot from 2024-08-15 16-13-39

### PR DESCRIPTION
**Pull request by DanielleMaywood**
Fixes https://github.com/coder/coder/issues/14315

This adds a 'Groups' column to the table on the `organizations/<name>/members` page.

Before change

Screenshot 2024-09-09 at 16 29 39

After change

Screenshot 2024-09-09 at 16 25 44

### CODE REVIEW

File: site/src/pages/ManagementSettingsPage/OrganizationMembersPage.tsx
Comment (aslilac) [inline site/src/pages/ManagementSettingsPage/OrganizationMembersPage.tsx:26]: note how this query doesn't take the `organizationName` as a parameter: this is not scoped to an organization, this shows _every_ group a user is in.

**Review by aslilac — CHANGES_REQUESTED** [review]

**Review by BrunoQuaresma — APPROVED** [review]
Code looks good to me but it is best to wait for the Kayla's approval 👍

**Review by aslilac — APPROVED** [review]
beautiful! thank you!

**Pull request merged**
